sds_data_manager/lambda_code/SDSCode/pipeline_lambdas/batch_starter.py

```diff
@@ -293,6 +293,60 @@ def find_most_recent_start_date(dep: dict, date: datetime) -> datetime:
     return new_start_date, new_end_date
 
 
+def duplicate_job(
+    instrument, data_level, descriptor, start_date, version, serialized_dependencies
+) -> bool:
+    """Determine if the current job is a duplicate of the most recent job.
+
+    Parameters
+    ----------
+    instrument : str
+        Instrument.
+    data_level : str
+        Data level.
+    descriptor : str
+        Data descriptor.
+    start_date : str
+        Start date.
+    version : str
```

Comment:
Can you change this to `previous_version`? This will help reduce confusion with versions from other functions.
```suggestion
    version : str
```